sequel/index.js

```diff
@@ -188,7 +188,7 @@ Sequel.prototype.update = function update(currentTable, queryObject, data) {
   };
 
   // Get the attribute identity (as opposed to the table name)
-  var identity = _.find(_.values(this.schema), {tableName: currentTable}).identity;
+  var identity = currentTable;
```

Comment:
So we don't want to do the aliasing described in the comment line below?  If not, it seems like line 193 doesn't make sense any more.